.appveyor.yml

```diff
@@ -1,8 +1,15 @@
 environment:
   matrix:
-  - TARGET: x86_64-pc-windows-msvc
-  - TARGET: i686-pc-windows-msvc
-  - TARGET: i686-pc-windows-gnu
+    - TARGET: x86_64-pc-windows-msvc
+    - TARGET: i686-pc-windows-msvc
+    - TARGET: i686-pc-windows-gnu
+
+matrix:
+  allow_failures:
```

Comment:
Yes, that's right. I'm not sure if it would still indicate it in the UI, but it definitely wouldn't block us from merging. It'd be good to know if something breaks, but I'm personally not going to install a VM and do proper testing there.